frontend/src/components/rankingTeamList.js

```diff
@@ -122,7 +122,7 @@ class RankingTeamList extends Component {
             )}
             <td>{team.profile.auto_accept_ranked ? "Yes" : "No"}</td>
             <td>{team.profile.auto_accept_unranked ? "Yes" : "No"}</td>
-            {this.props.canRequest && (
+            {this.props.canRequest && !this.props.episode_info.frozen && (
```

Comment:
oh, this check should probably have been inside the `td` rather than outside

Reply:
wait so i messed around w the headers and nothing bad seemed to happen?
if oddities occur then we can change later

Reply:
Will be resolved by #440

Reply:
(It wasn't actually a problem originally to be fair, it was only bad when the td sometimes showed and sometimes didn't which wasn't the case here. But the new behaviour (disabling) should be consistent with accept/reject.)